### DISCUSSION TITLE
Looking forward to new versions

### DISCUSSION THREAD

**Discussion by jiaaom** [mrkai77/Loop#388] [Q&A]
Hi Loop Developers,

Thank you for improving the way I use my Mac. It boosted my productivity and is way better than the tiling feature shipped with the new macOS 15 beta. I cannot live without this amazing tool. I wonder when the next version will be released? I'm eagerly looking forward to it. Thank you!

**Comment by mrkai77** [comment]
Thank you so much! I've been working super hard over the past few weeks to remake Loop's preferences window (#361), and currently am focused on some more minor fixes before releasing the next version :D

If you'd like to try a development version, here's the latest build: [Loop.zip](https://github.com/user-attachments/files/15781824/Loop.zip)!

**Reply by jiaaom** [reply]
A million thanks!! I appreciate your hard work. The new version looks absolutely awesome.
